### ISSUE TITLE
add examples 

### ISSUE THREAD

**Issue by dkboon01** [dotnet/AspNetCore.Docs#21877]
[Enter feedback here]
Can someone add an example of the format for the scope should be?  For instance, if I'm using azure ad to get to an api is the format "api://dkskjfadkak-adkjakjd-/Read.All" or "https:\\domain.name\" and then show where in Azure AD this value is displayed.   Its so hard to follow with no examples.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 92b9c5cb-0e48-25bc-2b7e-e7354bafbe2d
* Version Independent ID: b1e8aedc-b46b-41f1-55c1-e293bc6c93db
* Content: [Secure an ASP.NET Core Blazor WebAssembly standalone app with Azure Active Directory](https://docs.microsoft.com/en-us/aspnet/core/blazor/security/webassembly/standalone-with-azure-active-directory?view=aspnetcore-5.0&source=docs#access-token-scopes)
* Content Source: [aspnetcore/blazor/security/webassembly/standalone-with-azure-active-directory.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/blazor/security/webassembly/standalone-with-azure-active-directory.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-blazor**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
Hello @dkboon01 ... This is because this is the most generic of the WASM security topics only directly relying upon AAD for authn/z, so the scope could be almost anything depending on the web API (i.e., Azure endpoints are but one of **_thousands_** of access token scopes that one might configure for an app). When the app is hosted, the web API is in the server-side ASP.NET Core app in Azure, and we **_DO_** provide examples and notes at ...

https://docs.microsoft.com/aspnet/core/blazor/security/webassembly/hosted-with-azure-active-directory?view=aspnetcore-5.0#access-token-scopes

There's also info for Graph API scopes in the topic we have on that subject at ...

https://docs.microsoft.com/aspnet/core/blazor/security/webassembly/graph-api?view=aspnetcore-5.0

Try those sections and see if they help.

In the meantime, I'm going to use your issue here to cross-link the first one :point_up: to the section of this topic ... I'll add it in the bullet list of the existing two cross-links there. That probably would have helped you.

Leave this issue open. It will close automatically when the PR merges.
